index.css

```diff
@@ -0,0 +1,457 @@
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/fonts/Lato/Lato-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/fonts/Lato/Lato-Regular.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/fonts/Lato/Lato-Light.ttf');
+  font-weight: lighter;
+  font-style: normal;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --color-blue: hsl(214, 50%, 50%);
+  --color-grey: hsl(213, 16%, 22%);
+  --color-white: hsl(0, 0%, 100%);
+  --color-green: #516B5E;
+  --color-pink: #DD7F77;
+  --color-beige: #F8F3DF;
+  --color-diabled: #575654;
+  --color-background: hsla(213, 16%, 22%, .75);
+  --main-shadow: 0 0 1.5rem 0 var(--color-grey);
+  --text-large: calc(var(--text-body) * 2.25);
+  --text-medium: calc(var(--text-body) * 1.75);
+  --text-small: calc(var(--text-body) * 1.375);
+  --text-body: 1.6rem;
+  --text-xs: 1.2rem;
+}
+
+html,
+body {
+  width: 100vw;
+  height: 100%;
+  margin: 0;
+}
+
+* {
```

Comment:
You can also apply these rules to `html` instead of `*` (= little performance increase)